fhir-profile/src/main/java/com/ibm/fhir/profile/ExtensionBuilder.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.fhir.profile;
```

Comment:
I put this one next to ProfileBuilder under `src/main`, but I'm wondering if we should move them both to `src/test`...any thoughts?

Reply:
since it's hard coding the url, I think src/test is the right place